workflow-language/src/main/java/com/symphony/bdk/workflow/swadl/v1/activity/message/SendMessage.java

```diff
@@ -13,7 +12,7 @@
 @EqualsAndHashCode(callSuper = true)
 @Data
 public class SendMessage extends BaseActivity {
-  private String content;
+  private Object content;
```

Comment:
Could we override the setter and have a String content field and a String template field (with the setter assigning the proper field?)